cmd/gator/gator.go

```diff
@@ -13,6 +14,7 @@ const version = "alpha"
 var commands = []*cobra.Command{
 	verify.Cmd,
 	test.Cmd,
```

Comment:
We probably want to update `gator test` to also be expansion aware, since it helps with shift-left enforcement.